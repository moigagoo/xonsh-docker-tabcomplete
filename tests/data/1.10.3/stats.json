{
        "arguments": [
                "-a", "--all",
                "--help",
                "--no-stream"
        ],
        "commands": []
}
