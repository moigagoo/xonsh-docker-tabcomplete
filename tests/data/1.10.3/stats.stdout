
Usage:	docker stats [OPTIONS] [CONTAINER...]

Display a live stream of container(s) resource usage statistics

  -a, --all          Show all containers (default shows just running)
  --help             Print usage
  --no-stream        Disable streaming stats and only pull the first result
